src/components/deploymentConfig/DeploymentConfig.tsx

```diff
@@ -382,6 +418,10 @@ export default function DeploymentConfig({
                 type: DeploymentConfigStateActionTypes.fetchedValues,
                 payload: {},
             })
+            dispatch({
```

Comment:
We should use multipleOption action update both in single dispatch

Reply:
done